includes/Checker/Abstract_Check_Runner.php

```diff
@@ -74,6 +74,14 @@ abstract class Abstract_Check_Runner implements Check_Runner {
 	 */
 	protected $runtime_environment;
 
+	/**
+	 * Whether to include experimental checks.
+	 *
+	 * @since n.e.x.t
+	 * @var bool
+	 */
+	protected $include_experimental = false;
```

Comment:
See my other comment, this should probably be `null` initially and then be populated via `get_include_experimental_param()` if needed.

Reply:
Updated